github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml

```diff
@@ -116,6 +116,34 @@ presubmits:
               memory: "8Gi"
             limits:
               memory: "8Gi"
+  - name: build-bootstrap-legacy-image
+    always_run: false
+    run_if_changed: "images/bootstrap-legacy/.*|images/golang-legacy/.*"
+    decorate: true
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-kubevirtci-quay-credential: "true"
```

Comment:
no need here credential

Reply:
It shouldn't need those credentials - again a carry over from the existing bootstrap. Maybe better to address both in a follow on PR.

Reply:
This one can be removed on this PR basically, but whatever you prefer

Reply:
Yes you are right. Will remove it here.